bionic/module_reloader.py

```diff
@@ -0,0 +1,24 @@
+import builtins
```

Comment:
super nit: `module_reloader` is a bit of an unusual name for a module; usually they're named after concepts (like `persistence` or classes (like `flow`). In this case, a natural name might be `reloading`.

Reply:
Renamed to `reload.py`